test/api/buffer/import/import/expected-etcd-kvs.txt

```diff
@@ -186,6 +186,6 @@
   "sliceId": "%s",
   "count": 2,
   "size": %d,
-  "lastReceivedAt": "%s"
+  "lastAt": "%s"
```

Comment:
Kedze `Stats` je/bude spolocne aj pre ine statistiky, tak zmena v nazve kluca.

Reply:
🤔 Takže to je timestamp té zaznamenané události? Zní to trochu podivně, ale asi budiž. 😄

Reply:
no ja to citam tak ... key je `/stats/received/...` a vo value `lastAt` = teda `last received at`

podobne key je `/stats/imported/...` a vo value `lastAt` = teda `last imported at`

Mozem to nazvat inak, ak mas napad ako sa to moze volat nejak vseobecne pre tieto ^^^ pripady.

Reply:
@JakubMatejka este mi napada `lastRecordAt`?

Reply:
Chtěl jsem říct ať to necháš, ale `lastRecordAt` mi přijde dobrý. 🙂

Reply:
https://github.com/keboola/keboola-as-code/pull/1091/commits/73d668ff5d1dbf99c741b7754c00abc444ca2b22